### ISSUE TITLE
bourbon/neat + libsass compile issue

### ISSUE THREAD

**Issue by crispconcepts** [bdkjones/CodeKit2#474]
CodeKit: 2.1.8
Lang: scss
Compiler: libsass

Details:
When using libsass as the compiler, I receive the following error.

```
/Applications/CodeKit.app/Contents/Resources/engines/neat/functions/private:37: error: non-terminal statement or declaration must end with ';'
```

According to the team over at thoughtbot, it may be an issue with the libsass version you're using.
https://github.com/thoughtbot/neat/issues/255

scss:

```
@import "bourbon", "neat";
```

**Comment by bdkjones** [comment]
Yep. Sass is currently a nightmare. 

Zurb Foundation won't update to support the latest release of Libass, other frameworks are dropping support for the older version and Ruby Sass and Libsass conflict with each other all over the place.

I'll be issuing an update tonight or tomorrow with Libsass 3.

**Comment by bdkjones** [comment]
This should be fixed in the upcoming 2.2 release, which has new versions of everything.
